llvm/utils/release/github-upload-release.py

```diff
@@ -77,20 +77,23 @@ def upload_files(repo, release, files):
 parser.add_argument("--token", type=str)
 parser.add_argument("--release", type=str)
 parser.add_argument("--user", type=str)
+parser.add_argument("--user-token", type=str)
 
 # Upload args
 parser.add_argument("--files", nargs="+", type=str)
 
 args = parser.parse_args()
 
-github = github.Github(args.token)
-llvm_org = github.get_organization("llvm")
+gh = github.Github(args.token)
+llvm_org = gh.get_organization("llvm")
 llvm_repo = llvm_org.get_repo("llvm-project")
+if not args.user_token:
```

Comment:
Why do we need this here if we only support using the `--token` flag in combination with the `--user-token` flag here? Shouldn't we be erroring out rather than setting one to the other?

Reply:
Ah, didn't consider local testing. But yes, that was my suggestion. If it's for local testing, it shouldn't be a big deal to leave it in, but having a comment explaining what this is for would be good.

I think it would probably still be cleaner to just require `--user-token` if  `--user` is specified and pass both tokens during testing if needed, but no strong opinions there.